src/lib/utilities.js

```diff
@@ -63,3 +63,30 @@ export const concatenateUint8Arrays = (...arrays) => {
   }, 0);
   return result;
 };
+
+export const arrayBufferToHex = buffer => {
+  if (buffer.buffer instanceof ArrayBuffer && buffer.byteLength !== undefined) {
+    throw new TypeError("Expected input to be an ArrayBuffer");
+  }
+  return Array.prototype.map
+    .call(new Uint8Array(buffer), x => `00${x.toString(16)}`.slice(-2))
+    .join("");
+};
+
+export const hexToArrayBuffer = hex => {
```

Comment:
where do you use this function?

Reply:
I wrote a simple test to validate that the function `arrayBufferToHex` works as expected

Reply:
then it shouldn't be part of the lib, only part of the tests